### PR TITLE
feat(Button): add `floated` prop to Button.Group

### LINKED ISSUE #1245: feat(Button): add `floated` prop to Button.Group
`<Button.Group floated='right' />` works in SUI core, however, we are not applying the floated prop to groups in SUIR.

It should output:

```html
<div class="ui right floated buttons"></div>
```

### PR DESCRIPTION
**Pull request by mohammedkhadar**
This pull request fixes #1245

### CODE REVIEW

File: test/specs/elements/Button/ButtonGroup-test.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ describe('ButtonGroup', () => {
   common.implementsWidthProp(ButtonGroup, { propKey: 'widths', widthClass: 'buttons', canEqual: false })
 
   common.propKeyAndValueToClassName(ButtonGroup, 'attached')
+  common.propKeyAndValueToClassName(ButtonGroup, 'float')
```
Comment (levithomason) [inline test/specs/elements/Button/ButtonGroup-test.js]: Small typo, "floated".

**Review by levithomason — CHANGES_REQUESTED** [review]

**Comment by levithomason** [comment]
Thanks for the PR. It would be fantastic to get a doc example for this prop as well!

File: test/specs/elements/Button/ButtonGroup-test.js (hunk 1 above)
Comment (mohammedkhadar) [reply test/specs/elements/Button/ButtonGroup-test.js]: Typo fixed.

**Comment by levithomason** [comment]
Thanks much!

**Comment by layershifter** [comment]
We forget typings there, I'll make PR soon.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.64.5`.

**Pull request merged**